distribution/lib/Standard/Test/0.0.0-dev/src/Bench.enso

```diff
@@ -1,7 +1,68 @@
 from Standard.Base import all
 import Standard.Base.Runtime.Ref.Ref
 
+
+type Bench_Options
+    ## PRIVATE
+    Impl iter_size num_iters
```

Comment:
This might be a bit unrelated question. But I have never understood why is there need for both `iter_size` and `num_iters`? What is their precise meaning? Why not only `num_iters`? cc @radeusgd

Reply:
The PR only mimics what was already present in `Bench.measure`. It is not goal of this PR to improve runtime aspects (like warmup) of the benchmarks. I don't know what `iter_size` is supposed to mean, but I don't care (in this PR).

The answer doesn't really changes the ability to find and execute _just a single benchmark_ - which is the goal of this PR.

Reply:
I don't know precisely either 😅 I imagine @kustosz would know this.

What happens is `num_iterations` drives how many lines we will get in the output. `iter_size` says how many runs are done in a single iteration to then compute an average of these for the one line of output. I imagine it increasing `iter_size` could allow to reduce the variance of a measurement.

However, personally I'd rather prefer `iter_size=1` and then having more detailed datapoints.

Unless there are good reasons to have it, I'm happy to remove it and keep `num_iters` only.